src/test/java/core/basesyntax/service/fileservice/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.service.fileservice;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderServiceImplTest {
+    private static FileReaderService fileReader;
+    private static String validFilePath;
+    private static String invalidFilePath;
+    private static List<String> expectedList;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fileReader = new FileReaderServiceImpl();
+        expectedList = new ArrayList<>();
+        validFilePath = "src/test/resources/testInput.csv";
+        invalidFilePath = "notExistingPath";
+        expectedList.add("type,fruit,quantity");
+        expectedList.add("b,banana,20");
+        expectedList.add("b,apple,100");
+        expectedList.add("s,banana,100");
+        expectedList.add("p,banana,13");
+        String lines = "type,fruit,quantity\n"
+                + "b,banana,20\n"
+                + "b,apple,100\n"
+                + "s,banana,100\n"
+                + "p,banana,13";
+        File file = new File(validFilePath);
+        try {
+            Files.writeString(file.toPath(), lines);
```

Comment:
I meant that we can create this file not programmatically, 
but by right click -> create file and copy data that we need to that file
<img width="770" alt="image" src="https://user-images.githubusercontent.com/58032907/200560662-7f99e496-770a-4a9f-a555-47b46ab375c0.png">